README.md

```diff
@@ -1,8 +1,107 @@
 # ActiveRecord Associations in Sinatra
 
-##TEACHER OUTLINE
-+ Creating multiple tables
-+ Describe has many/belongs to relationships
-+ Setting up has_many, belongs_to relationship between models
-+ Discuss the use of primary keys as the link between models in ActiveRecord
-+ Should allude to other types of relationships (many to many, one to one, etc) but not discuss join tables.
\ No newline at end of file
+## Objectives
+
+1. Understand primary and foreign key.
+2. Understand the has many/ belongs to relationship.
+
+
+### Primary Keys
+
+Let's assume we have two tables in our database `cats` and `owners`, which we created from the command line using rake.
```

Comment:
Is this a codealong (it should be)?  If so, can we get the student acquainted with the set up - ie. walk them through the tables and have them run rake db:migrate.